app/api/order_statistics/tickets.py

```diff
@@ -99,7 +99,7 @@ def sales_count(self, obj):
         placed = db.session.query(func.sum(Order.amount.label('sum'))).join(Order.order_tickets).filter(
             OrderTicket.ticket_id == obj_id, Order.status == 'placed').scalar()
         completed = db.session.query(func.sum(Order.amount.label('sum'))).join(Order.order_tickets).filter(
-            OrderTicket.ticket_id == obj_id, Order.status == 'completed').scalar()
+            OrderTicket.ticket_id == obj_id, Order.status == 'completed' or Order.status == 'placed').scalar()
         result = {
             'total': total or 0,
             'draft': draft or 0,
```

Comment:
@kushthedude I agree to @uds5501 
We should apply simple addition on FE to show the stats. I don't see any difficulty or problem here.
Why exactly will this create a confusion?
We should leave the `Completed` and `Placed` stats as it is on server IMO